### ISSUE TITLE
when epoch = 1 ,the loss of Reppoints turn very small

### ISSUE THREAD

**Issue by xjl-le** [open-mmlab/mmdetection#3207]
Thanks for your error report and we appreciate it a lot.

**Checklist**
1. I have searched related issues but cannot get the expected help.
2. The bug has not been fixed in the latest version.

**Describe the bug**
when I train reppoint,I found I can't download Pre-training file ResNet50,what'more.when epoch =1，the loss of reppoints turn to be 0.69。I know it is wrong,but I can't moderfied anything 。
**Reproduction**
1. What command or script did you run?
python train.py
A placeholder for the command.
--config  configs/reppoints/reppoint_r50_moment_fpn_gn_neck-head_1x_coco.py
2. Did you make any modifications on the code or config? Did you understand what you have modified?
no
3. What dataset did you use?
coco
**Environment**
pytorch 1.3 cuda10
1. Please run `python mmdet/utils/collect_env.py` to collect necessary environment infomation and paste it here.
2. You may add addition that may be helpful for locating the problem, such as
    - How you installed PyTorch [e.g., pip, conda, source]
pip 
    - Other environment variables that may be related (such as `$PATH`, `$LD_LIBRARY_PATH`, `$PYTHONPATH`, etc.)

**Error traceback**
If applicable, paste the error trackback here.

![error](https://user-images.githubusercontent.com/55579267/86527343-a5103800-bed0-11ea-930f-4e2ce7f8512a.png)

A placeholder for trackback.
```

**Bug fix**
If you have already identified the reason, you can provide the information here. If you are willing to create a PR to fix it, please also leave a comment here and that would be much appreciated!

**Comment by ZwwWayne** [comment]
Why you think it is wrong? You can refer to the [log](https://open-mmlab.s3.ap-northeast-2.amazonaws.com/mmdetection/v2.0/reppoints/reppoints_moment_r50_fpn_1x_coco/reppoints_moment_r50_fpn_1x_coco_20200330_233609.log.json), from the log it seems that your training is normal.
